app/src/main/java/com/globant/example/mentorapp/mvp/base/BaseFragment.java

```diff
@@ -0,0 +1,21 @@
+package com.globant.example.mentorapp.mvp.base;
+
+import android.arch.lifecycle.LifecycleFragment;
+
+import com.globant.example.mentorapp.data.livedata.SharedViewModel;
+
+/**
+ * Generic class to define basic fragment behavior
+ * Created by oscar.amezquita on 4/08/2017.
+ */
+
+public class BaseFragment extends LifecycleFragment {
+
+    protected BaseActivity parent;
+    protected SharedViewModel model;
+
+    protected void showErrorMessage(String message) {
+        parent.simpleSnackBarMessage(message);
```

Comment:
Can it be null?

Also, you can initialize these attributes in this class if the logic applies for all the child classes